Vagrantfile

```diff
@@ -1,6 +1,27 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+require 'getoptlong'
+
+hostmem = %x[ grep MemTotal /proc/meminfo ].scan(/\d+/).shift.to_i / 1024
```

Comment:
talked to @hennevogel:
WIll switch to environment variables OBS_VAGRANT_CPU / OBS_VAGRANT_MEM which could be overwritten with the mem and cpu cli options